gcp/variables.tf

```diff
@@ -16,6 +16,12 @@ variable "kibana_server_port" {
   description = "Kibana's dev server port (5601 unless you have explicitly changed it in `kibana.dev.yml`)."
 }
 
+variable "kibana_server_password" {
```

Comment:
should add var to `terraform.tfvars.example` as well

Reply:
very true! thx!

Reply:
done